### DISCUSSION TITLE
Cannot encrypt anymore

### DISCUSSION THREAD

**Discussion by hammaschlach** [cupcakearmy/cryptgeon#49] [Q&A] [answered]
Hi,

i'm running cryptgeon on my synology nas.
When clicking on "create" i get an error: "file cannot be created. try later"

<img width="140" alt="Bildschirmfoto 2022-07-26 um 10 04 01" src="https://user-images.githubusercontent.com/69586333/180972761-7ab473be-e4b1-453c-8de6-001d71975257.png">

**Comment by cupcakearmy** [comment, marked as answer]
Moin moin, 
Hast du noch fehlermeldungen dazu?
Ist es eine textnotiz oder eine datei?

**Reply by hammaschlach** [reply]
Hallo,

nein, das ist die einzige Meldung, die ich erhalte.
Es ist unabhängig davon, ob Text oder Datei.

**Reply by cupcakearmy** [reply]
Kannst du mal gucken was für versionen laufen und im backend gibt es keine logs?
Die neuste version benutz redis und nicht mehr memecace, vielleicht ist das ein problem?

**Reply by hammaschlach** [reply]
Ich habe gerade per Portainer "frisch" gem. https://github.com/cupcakearmy/cryptgeon installiert (2.0.2).
Der Fehler ist leider geblieben. Logs sind keine zu finden. Es gäbe lediglich einen Terminalzugang zu dem Container.
Ich muss dazu sagen, was Docker angeht bin ich absoluter Neuling.

**Reply by hammaschlach** [reply]
Nun haut es hin. Wenn ich per Reverse Proxy von „außen“ zugreife, funktioniert es. Wenn ich lokal (mit Angabe des Ports) versuche, kommt es zu dem Fehler.

**Reply by cupcakearmy** [reply]
Gibt es da was fertiges? Kenn mich mit synology nicht aus. habe nur das [hier gesehen gerade](https://mariushosting.com/how-to-install-cryptgeon-on-your-synology-nas/). Da wird aber auch ein stack angegeben (step 5). Sicher das da schon redis drinne ist?

**Reply by hammaschlach** [reply]
An der Anleitung hatte ich mich entlang gehangelt. Ich werde den Autor auf redis hinweisen.

**Reply by cupcakearmy** [reply]
Hab ihm schon geschrieben :)
